test/support/utils.rb

```diff
@@ -59,7 +60,7 @@ def create_pseudo_terminal(boot_options: "-r debug/run")
             while (line = read.gets)
               debug_print line
               case line.chomp
-              when '(rdbg)'
+              when REPL_PROMPT
```

Comment:
I'm sorry for commenting even though I approved.

If REPL ERROR is in test program, does it stop?

Reply:
do you mean the Ruby script in `def program`? why would we put `REPL ERROR` there?
if it's printed by the debugger from pseudo terminal, it' will stop (which is the purpose of this PR).

Reply:
> do you mean the Ruby script in def program?
Yes.

Thanks, I just wanted to know about it.

Reply:
well yes. since it doesn't understand the difference between output content, it'll match the program's content (like `s = "REPL ERROR"`).